infrasim/model.py

```diff
@@ -628,13 +628,14 @@ def __init__(self):
         self.__wwn = None
         self.__bootindex = None
         self.__version = None
+        self.__share_rw = False
```

Comment:
What if setting default to true? it does no harm to leave it to be true and that drive is accessed by only one host, right?

Reply:
One of our functional test yml configure is 2 drive sharing 1 disk, which need the share_rw to be true. If default is false, need to update that test.
Anyway, set the default value "true" doesn't harm. Right?

Reply:
I'll set "share-rw" to true in func test.

Reply:
@fub2 , @JuneZhou19 no problem~2 vote on default true
I'm OK with that